Dockerfiles/basic-build-environments/debian_jessie/checkout_and_build_syslog_ng_incubator.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -xe
+
+if [ ${BRANCH_NAME}=master ] ; then
+  git clone -b ${INCUBATOR_BRANCH_NAME} https://github.com/balabit/syslog-ng-incubator.git /syslog-ng-incubator-src
+  cd /syslog-ng-incubator-src
+  git pull
+  autoreconf -i
+  mkdir build_dir && cd build_dir && mkdir -p ${INSTALL_DIR}
+  ../configure \
+    --prefix=${INSTALL_DIR}
+  make
+  make install
+else
+  echo "syslog-ng OSE incubator not supported with this OSE version: ${BRANCH_NAME}, it will be not downloaded, installed."
```

Comment:
That's ok. I don't know what should be the result of the build. We don't support other branches, so we may  execute `false` to stop the building immediately. We may build "empty" images without a failing command, or I'm missing something.